tests/01http-server.pl

```diff
@@ -5,6 +6,9 @@
 use SyTest::HTTPClient;
 use SyTest::HTTPServer::Request;
 
+require IO::Async::SSL;
```

Comment:
'require' is fine here but for consistency with all the others above, might as well 'use' it.
Also needs blank line below.